ext/nnstreamer/tensor_decoder/tensordecutil.h

```diff
@@ -36,6 +38,8 @@ initSingleLineSprite (singleLineSprite_t v, rasters_t r, uint32_t pv);
 
 extern void _free_labels (imglabel_t *data);
 
+extern void capsConfigFrameRate (GstCaps *caps, const GstTensorsConfig * config);
```

Comment:
Please rename the function for readability:

```setFramerateFromConfig ( caps, config )```

Reply:
Thanks for a better name :) I will change it.